Makefile.in

```diff
@@ -349,7 +349,7 @@ libaddon: exports
 	$(MAKE) -C lib/addons/library.kodi.inputstream
 addon-dev-kit:
 	mkdir -p xbmc/addons/kodi-addon-dev-kit/build
-	cd xbmc/addons/kodi-addon-dev-kit/build; $(CMAKE) -DCMAKE_INSTALL_PREFIX=$(DESTDIR)$(prefix) ..
+	cd xbmc/addons/kodi-addon-dev-kit/build; cmake -DCMAKE_INSTALL_PREFIX=$(DESTDIR)$(prefix) ..
```

Comment:
I am currently trying to find a way to bring up cmake of "native".

Wanted now first be sure that it is found. But that built does not yet what it should be.